doc/nvim-tree-lua.txt

```diff
@@ -273,6 +273,7 @@ Setup may only be run once; subsequent calls will result in a warning.
       filesystem_watchers = {
         enable = false,
         interval = 100,
+        debounce = 50,
```

Comment:
This doesn't really matter for now; we can tune it when we see how fast fs_event fires.